app/helpers/facets_helper.rb

```diff
@@ -0,0 +1,27 @@
+module FacetsHelper
+  # override host's tabs to add all facets as tabs
+  def host_additional_views(host)
+    base_tabs = load_facets(host)
+    base_tabs.merge!(helper_tabs(host))
+    base_tabs
+  end
+
+  def load_facets(host)
+    Hash[host.host_facets_with_definitions.map do |facet, definition|
+      [definition.name, facet]
+    end]
+  end
+
+  def helper_tabs(host)
+    tab_definitions = {}
+    host.host_facets_with_definitions.each do |facet, facet_definition|
+      if facet_definition.tabs.is_a? Hash
+        tab_definitions.merge!(facet_definition.tabs)
+      else
+        method_result = send(facet_definition.tabs, host)
```

Comment:
This seems to indicate that if i pass  :my_facet_tabs as a tab, that this will call  `send(:my_facet_tabs)` implying that that method is defined in my helper that i have also specified (:my_facet_helper), but i don't see where that helper would be 'included'.  When i try to use this way of doing tabs i get undefined method because the 'my_facet_tabs' method doesn't exist